### PR TITLE
Tweak linux cheatsheet

### PR DESCRIPTION
**Pull request by NaveenKarippai**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.

- [ ] New Instant Answer
    - [ ] Cheat Sheets: **`New {Cheat Sheet Name} Cheat Sheet`**
    - [ ] Other: **`New {IA Name} Instant Answer`**
- [x] Improvement
    - [ ] Bug fix: **`{IA Name}: Fix {Issue number or one-line description}`**
    - [x] Enhancement: **`linux_cheat_sheet: meta-data, description, title and listed commands modified`**
- [ ] Non–Instant Answer
    - [ ] Other (Role, Template, Test, Documentation, etc.): **`{GoodieRole/Templates/Tests/Docs}: {Short Description}`**

###### Description of changes

There would be users who might search for aliases such as
gnu linux cheatsheet. so, aliases would be necessary meta-data
to this cheatsheet. There could also be minor modifications to some
commands listed.

Provide an overview of the changes this pull request introduces.

* alias meta-data added

* description and title modified

* commands modified

###### Which issues (if any) does this fix?

Fixes #NNNN - how specifically does it fix the issue?

###### People to notify (@GuiltyDolphin)

---

Instant Answer Page: https://duck.co/ia/view/linux_cheat_sheet

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @crashrane

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by NaveenKarippai** [comment]
@GuiltyDolphin could you please make a comment why CI failed for 3dc7f4d on this PR. 

The JSON data seems to be valid + test passes locally (on CI build too).

**Comment by jophab** [comment]
Lets have the capitalization of first letter in val field consistent

**Comment by NaveenKarippai** [comment]
@jophab Thanks for the review! Could you please go through the new changeset :+1: 

@GuiltyDolphin just a suggestion: how about adding **guidelines section on cheatsheet documentation** to follow capitalisation and remove special characters (periods) in JSON data? This could help to make sure that cheatsheets looks more consistent.

**Comment by moollaza** [comment]
Thanks! This LGTM :+1:

File: share/goodie/cheat_sheets/json/linux.json [hunk 1]
```diff
@@ -22,263 +26,283 @@
...
             {
-                "val": "change the current directory to the root directory",
+                "val": "Change the current directory to the root directory",
                 "key": "cd /"
             },
             {
-                "val": "change to the parent of the current directory",
+                "val": "Change to the parent of the current directory",
                 "key": "cd .."
             },
```
Comment (laorynas) [inline share/goodie/cheat_sheets/json/linux.json]: ## I think you should add this to the list.

`"val": "Listing of all files with names ending in ".jpg"`
`"key": "ls *.jpg"`

`"val": "Listing of all files with names beginning with "a"`
`"key": "ls a*"`

`"val": "Listing of all files with names containing "lin"`
`"key": "ls *lin*"`

File: share/goodie/cheat_sheets/json/linux.json [hunk 2]
```diff
@@ -22,263 +26,283 @@
...
             {
-                "val": "list directory contents",
+                "val": "List directory contents",
                 "key": "ls"
             },
             {
-                "val": "formatted long listing with hidden files",
-                "key": "ls -al"
+                "val": "Formatted long listing with hidden files",
+                "key": "ls -alh"
```
Comment (laorynas) [inline share/goodie/cheat_sheets/json/linux.json]: ## I think you should add this to the list.

`"val": "Listing of all files with names ending in ".jpg"`
`"key": "ls *.jpg"`

`"val": "Listing of all files with names beginning with "a"`
`"key": "ls a*"`

`"val": "Listing of all files with names containing "lin"`
`"key": "ls *lin*"`

**Pull request merged**